Jenkinsfile

```diff
@@ -11,48 +11,67 @@
 
 // TEST FLAG - to make it easier to turn on/off unit tests for speeding up access to later stuff.
 def runTests = true
+def failFast = false
 
 // Only keep the 10 most recent builds.
 properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator',
                                                                         numToKeepStr: '50',
                                                                         artifactNumToKeepStr: '20']]])
 
-node('java') {
-    timestamps {
-        // First stage is actually checking out the source. Since we're using Multibranch
-        // currently, we can use "checkout scm".
-        stage('Checkout') {
-            checkout scm
-        }
+def buildTypes = ['Linux', 'Windows']
 
-        // Now run the actual build.
-        stage("Build / Test") {
-            timeout(time: 180, unit: 'MINUTES') {
-                // See below for what this method does - we're passing an arbitrary environment
-                // variable to it so that JAVA_OPTS and MAVEN_OPTS are set correctly.
-                withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m",
-                            "MAVEN_OPTS=-Xmx1536m -Xms512m -XX:MaxPermSize=1024m"]) {
-                    // Actually run Maven!
-                    // The -Dmaven.repo.local=${pwd()}/.repository means that Maven will create a
-                    // .repository directory at the root of the build (which it gets from the
-                    // pwd() Workflow call) and use that for the local Maven repository.
-                    sh "mvn -Pdebug -U clean install ${runTests ? '-Dmaven.test.failure.ignore=true' : '-DskipTests'} -V -B -Dmaven.repo.local=${pwd()}/.repository"
+def builds = [:]
+for(i = 0; i < buildTypes.size(); i++) {
```

Comment:
Correct, pipeline groovy doesn't support nice things like .each and so forth right now as far as I know.